syntax = "proto3";

message LocationMessage {
  int64 id = 1;
  int64 person_id = 2;
  string longitude = 3;
  string latitude = 4;
  string creation_time = 5;
}

message RetrieveLocationRequest {
  int64 id = 1;
}

message Empty {
}

service LocationService {
  rpc Create(LocationMessage) returns (Empty);
  rpc Retrieve(RetrieveLocationRequest) returns (LocationMessage);
}